src/ToDo.UI/Views/SettingsPage.xaml

```diff
@@ -94,11 +94,15 @@
 									   Text="Mode"
 									   utu:AutoLayout.CounterAlignment="Start"
 									   Style="{StaticResource MaterialSubtitle1}" />
-							<utu:ChipGroup SelectionMode="Single"
+							<utu:ChipGroup x:Name="ThemeChipGroup"
+										   SelectionMode="Single"
+										   ItemChecked="{x:Bind UpdateAppTheme}"
```

Comment:
ooo neat

Reply:
@kazo0 thanks to @Xiaoy312 for his advices